ledger/src/shred.rs

```diff
@@ -117,9 +117,9 @@ const SIZE_OF_SIGNATURE: usize = SIGNATURE_BYTES;
 // data shreds per each batch as below. The actual number of data shreds in
 // each erasure batch depends on the number of shreds obtained from serializing
 // a &[Entry].
-pub const DATA_SHREDS_PER_FEC_BLOCK: usize = 32;
-pub const CODING_SHREDS_PER_FEC_BLOCK: usize = 32;
-pub const SHREDS_PER_FEC_BLOCK: usize = DATA_SHREDS_PER_FEC_BLOCK + CODING_SHREDS_PER_FEC_BLOCK;
+pub const DATA_SHRED_PER_FEC_SET: usize = 32;
```

Comment:
can not rename DATA_SHREDS_PER_FEC_BLOCK, it is public API break. others should be ok since we've added them like 3 days ago.